GolfExample_TCAPI/index.html

```diff
@@ -156,6 +240,26 @@
 
             return newValue;
         }
+        
+        function saveBookmarking(callback){
```

Comment:
This is kind of making some horrible use of globals that normally I'd definitely not let fly, but since this has languished for months I'm okay to let it through to get the PR out. Do we have the capacity to update this to make it better, or should we just be sad with the knowledge that it exists and move on?